lib/stdlib/python/Queue.py

```diff
@@ -0,0 +1,242 @@
+"""A multi-producer, multi-consumer queue."""
+
```

Comment:
For these files that are copied over from stdlib, we should reflect their provenance and copyright with e.g. a header comment describing the source of the file and the changes we've made, or a LICENSE.txt in the containing directory.